geonode/upload/admin.py

```diff
@@ -60,21 +60,10 @@ def clean(self):
 
         after_upload_slugs_list = ['dataset_upload_size', 'document_upload_size']
 
-        if slug == 'file_upload_handler':
-            after_upload_sizes = UploadSizeLimit.objects.filter(
-                slug__in=after_upload_slugs_list
-            ).values_list('max_size', flat=True)
-            if after_upload_sizes and max_size <= max(after_upload_sizes) * 2:
-                raise forms.ValidationError(_(
-                    "To avoid errors, max size should be at least 2 times "
-                    "greater than the value of others size limits."
-                ))
+        max_size_before_upload = (max_size * 2) + 2097152
 
         if slug in after_upload_slugs_list:
-            handler_max_size = UploadSizeLimit.objects.filter(
-                slug='file_upload_handler'
-            ).values_list('max_size', flat=True)
-            if handler_max_size and max_size * 2 >= max(handler_max_size):
+            if max_size * 2 >= max_size_before_upload:
                 raise forms.ValidationError(_(
                     "To avoid errors, max size should be at least 2 times "
                     "smaller than the value of 'file_upload_handler'."
```

Comment:
I think we also don't need this validation anymore, since max_size_before_upload will be always calculated using the document_upload_size and dataset_upload_size

Reply:
By consequence the string in these files can be removed:
* geonode/locale/en/LC_MESSAGES/django.po:6592
* geonode/locale/fr/LC_MESSAGES/django.po:6730
* geonode/locale/pt/LC_MESSAGES/django.po:6818
* geonode/locale/pt_BR/LC_MESSAGES/django.po:6591
* geonode/upload/api/serializers.py:255